docs/user_guide/emummc/making_emummc.md

```diff
@@ -17,8 +17,47 @@
 
 -----
 
+### Making a nand backup
+
+
+!!! danger "Important"
+    A NAND backup is crucial. They can be used to restore the device to a working state in case of emergencies, and will be required in order to migrate to an EmuNAND setup in the near future.
+
+	Once the backup is finished, **keep it somewhere safe.** The best backup is the one you have but never need, and the worst backup is the one you need but never made. To save space, it's recommended to compress the end-result with a `.zip` file or something similar.
```

Comment:
"compress the end-result with a `.zip` file" -> "compress the end-result to a `.zip` file" imo